app/views/content/is-teaching-right-for-me/languages/_article.html.erb

```diff
@@ -0,0 +1,105 @@
+<div class="row inset">
+  <section class="col col-720">
+    <p>Being a modern languages teacher is rewarding as you can open young minds to the languages and cultures around the world.</p>
+  <p>There are 18 modern languages you could teach with the most popular being French, German and Spanish.</p>
+  <p>Tax-free  bursaries of £25,000 or scholarships of £27,000 are available for eligible trainee modern languages teachers.</p>
```

Comment:
```suggestion
  <p>Tax-free  bursaries of £25,000 or scholarships of £27,000 are available for eligible trainee modern languages teachers.</p>
```

I think this is misleading - the scholarships are only for French, German and Spanish?